assignment/tool/src/webapp/js/studentViewSubmission.js

```diff
@@ -1,4 +1,8 @@
 var ASN_SVS = ASN_SVS || {};
+var ASN = ASN || {};
+var ASN_TS_API = ASN_TS_API || {};
+
+window.i18nProgresa = [];
```

Comment:
I thought this variable name was a typo but I think it's the spanish translation? It's consistent across the file so I don't have concerns other than if we have a style/standard to not do that.

Reply:
**i18nProgresa** renamed to **i18nWlogTab**